### ISSUE TITLE
Method of Deleting books from Library

### ISSUE THREAD

**Issue by choyan** [babluboy/bookworm#24]
Currently, to Delete a book, we need to select a button, then the system will start selecting it. This behavior can be improved by the idea of how elementary is using it in its system. As this is a universal behavior across elementary, I think bookworm should have it too.

![screenshot from 2017-02-27 13 17 41](https://cloud.githubusercontent.com/assets/1283240/23352075/4b3d122c-fcef-11e6-9faf-4b54e0a36eff.jpeg)

**Comment by babluboy** [comment]
Yes, I had tried to achieve the elementary behaviour - on mouseover the option of select shows and if the click is made on the top left corner then the selection is made. This seemed a bit complicated to achieve so I made the selection mode through a select button. Will try to achieve the elementary behaviour after the basic features like CSS, Font and Zoom are completed.
